channels/test/websocket.py

```diff
@@ -82,6 +82,9 @@ def send(self, to, content={}, text=None, path='/'):
         self.channel_layer.send(to, self._get_content(content, text, path))
         self._session_cookie = False
 
+    def _tuple_headers(self):
+        return [[x, self.headers[x]] for x in self.headers]
```

Comment:
Maybe `[[header, value] for header, value in self.headers.items()]` is more readable.

Reply:
Ill make those changes, but than the tests would still fail.

Reply:
You need to edit tests as well ;-)